tests/test_data/message/thread_missing_respondent.json

```diff
@@ -1,196 +1,106 @@
 {
-    "category": "SURVEY",
```

Comment:
This file was bloated and a bit of a mess, we only need 2 messages, one from the rops user and one from the respondent